app/src/main/java/org/gdg/frisbee/android/chapter/InfoFragment.java

```diff
@@ -120,14 +120,14 @@ public void onActivityCreated(Bundle savedInstanceState) {
 
         mInflater = LayoutInflater.from(getActivity());
 
+        final String chapterPlusId = getArguments().getString(Const.EXTRA_PLUS_ID);
```

Comment:
This can also be from the URI. Better to have a callback to the activity.
But different PR :-)